charts/argo-controller/templates/config.yaml

```diff
@@ -0,0 +1,33 @@
+---
+{{- $secretName := printf "%s-persistence-user" .Release.Name }}
+{{- $secretKey := "username" }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $secretName }}
+data:
+  {{ $secretKey }}: {{ .Values.persistence.username | b64enc | quote }}
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-config
+data:
+  config: |
+    containerRuntimeExecutor: pns
```

Comment:
TODO: This might be causing issues with pod security policies in GKE, might want to make it an enum with "docker" as the other option.